### PR TITLE
LPS-134654 Fix CSS selector to work in "older" version of Chrome

### PR DESCRIPTION
**Pull request by julien**
Our CI infrastructure is currently using Chrome 86, which doesn't
support the ["selector list argument of :not()"](https://caniuse.com/css-not-sel-list),
so here we're just making sure it works for that version.

**Test plan**
- Start up tomcat
- Use Google Chrome 86 :warning: 
- Navigate to http://localhost:8080
- Assert no errors are thrown in the console
- Assert that tooltips are working correctly

**Before**

**After**

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const SELECTOR_TRIGGER = `
 	.upper-tbar [data-title]:not(.lfr-portal-tooltip),
 	.upper-tbar [title]:not(.lfr-portal-tooltip),
 	.upper-tbar [data-restore-title],
-	.lfr-tooltip-scope [title]:not(iframe,[title=""]),
+	.lfr-tooltip-scope [title]:not(iframe):not([title=""]),
```
Comment (diegonvs) [inline modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx:35]: great catch 🎉

**Comment by matuzalemsteles** [comment]
So the use of the comma is just a new feature or was this really a bug specific to this version, do you have any idea @julien? Just for curiosity. Regardless, we can send this forward.

**Comment by bryceosterhaus** [comment]
@matuzalemsteles yeah it has to do comma separated selectors. See the [can i use](https://caniuse.com/?search=not%20selector)

**Comment by bryceosterhaus** [comment]
ci:forward

**Comment by matuzalemsteles** [comment]
oh thanks @bryceosterhaus for the info

**Comment by julien** [comment]
Thanks for forwarding guys! @matuzalemsteles, as @bryceosterhaus said yes, it's available in Chrome 88, I added a link in the message so we could remember it if something happens again.

**Comment by matuzalemsteles** [comment]
@julien perfect, thank you!

**Pull request closed without merging**